### PR TITLE
Spline Charts: update demo description

### PR DESCRIPTION
**Pull request by sergepilipchuk**

### CODE REVIEW

File: JSDemos/Demos/Charts/Spline/description.md [hunk 1]
```diff
@@ -1 +1,7 @@
...
\ No newline at end of file
+Spline charts, like [line charts](/Demos/WidgetsGallery/Demo/Charts/Spline/jQuery/Light/), allow you to visualize data and display them as lines with points placed over specified intervals. However, spline charts smooth out individual lines that connect data points. 
+
+In this example, you can switch through the Spline, Stacked Spline and Full-Stacked Spline charts. A Stacked Spline chart visualizes multiple data series and allows you to compare the manner in which each series contributes to the total aggregate value for specific arguments. A Full-Stacked Spline chart helps compare the percentage value of multiple line series for each argument. You can use the drop-down menu under the chart to switch between spline chart types mentioned above.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Charts/Spline/description.md]: - "to switch through" means [something different](https://idioms.thefreedictionary.com/switch+through).
- You have two sentences telling the reader that they can switch between chart types. Why don't we merge them?

```suggestion
In this example, you can use the drop-down menu under the chart to switch between the Spline, Stacked Spline and Full-Stacked Spline chart types. A Stacked Spline chart visualizes multiple data series and allows you to compare the manner in which each series contributes to the total aggregate value for specific arguments. A Full-Stacked Spline chart helps compare the percentage value of multiple line series for each argument.
```

File: JSDemos/Demos/Charts/Spline/description.md [hunk 2]
```diff
@@ -1 +1,7 @@
-Spline series types, like line series types, help visualize data trends over specified intervals. However, splines help &laquo;smooth&raquo; individual lines that connect data points. In&nbsp;this example, you can switch between the _spline_, _stacked spline_ and _full-stacked spline_ series types. _Stacked spline_ series help visualize multiple series and compare the manner in&nbsp;which each series contributes to&nbsp;the total aggregate value for specific arguments. _Full-stacked spline_ series help compare the percentage value of&nbsp;multiple spline series for each argument.
\ No newline at end of file
+Spline charts, like [line charts](/Demos/WidgetsGallery/Demo/Charts/Spline/jQuery/Light/), allow you to visualize data and display them as lines with points placed over specified intervals. However, spline charts smooth out individual lines that connect data points.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Charts/Spline/description.md]: Remove the suffix that specifies the framework and the theme. Our website stores the selected framework and theme in cookies and redirects the user to the corresponding version of a demo based on this information. If you keep the suffix, your link will point at a specific framework and a specific theme, which is rarely needed.

```suggestion
Spline charts, like [line charts](/Demos/WidgetsGallery/Demo/Charts/Line/), allow you to visualize data and display them as lines with points placed over specified intervals. However, spline charts smooth out individual lines that connect data points. 
```

File: JSDemos/Demos/Charts/Spline/description.md [hunk 3]
```diff
@@ -1 +1,7 @@
...
\ No newline at end of file
+Spline charts, like [line charts](/Demos/WidgetsGallery/Demo/Charts/Spline/jQuery/Light/), allow you to visualize data and display them as lines with points placed over specified intervals. However, spline charts smooth out individual lines that connect data points. 
+
+In this example, you can switch through the Spline, Stacked Spline and Full-Stacked Spline charts. A Stacked Spline chart visualizes multiple data series and allows you to compare the manner in which each series contributes to the total aggregate value for specific arguments. A Full-Stacked Spline chart helps compare the percentage value of multiple line series for each argument. You can use the drop-down menu under the chart to switch between spline chart types mentioned above.
+
+## Configure Series Type
+
+The series type defines the appearance of your chart. To display a Spline, Stacked Spline, or Full-Stacked Spline series, set the [type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#type) property to one of these types.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Charts/Spline/description.md]: We need to say clearly that for information about configuration the reader should go to the Line demo, with a note that for spline types the `type` property should be set to different values.

File: JSDemos/Demos/Charts/Spline/description.md [hunk 4]
```diff
@@ -1 +1,7 @@
...
\ No newline at end of file
+Spline charts, like [line charts](/Demos/WidgetsGallery/Demo/Charts/Line/), allow you to visualize data and display them as lines with points placed over specified intervals. However, spline charts smooth out individual lines that connect data points. 
+
+In this example, you can use the drop-down menu under the chart to switch between the Spline, Stacked Spline and Full-Stacked Spline chart types. A Stacked Spline chart visualizes multiple data series and allows you to compare the manner in which each series contributes to the total aggregate value for specific arguments. A Full-Stacked Spline chart helps compare the percentage value of multiple line series for each argument.
```
Comment (albertov05) [inline JSDemos/Demos/Charts/Spline/description.md]:
```suggestion
In this example, you can use the drop-down menu under the chart to switch between the Spline, Stacked Spline, and Full-Stacked Spline chart types. A Stacked Spline chart visualizes multiple data series and allows you to compare the manner in which each series contributes to the total aggregate value for specific arguments. A Full-Stacked Spline chart helps compare the percentage value of multiple line series for each argument.
```

File: JSDemos/Demos/Charts/Spline/description.md [hunk 5]
```diff
@@ -1 +1,7 @@
...
+
+## Configure Spline Charts
+
+The functionality of spline charts is very similar to that of line charts. You can use instructions from the [Line](/Demos/WidgetsGallery/Demo/Charts/Line/) demo description to configure spline charts. To display a Spline, Stacked Spline, or Full-Stacked Spline series, set the [type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#type) property to one of these types.
```
Comment (albertov05) [inline JSDemos/Demos/Charts/Spline/description.md]:
```suggestion
The functionality of Spline charts is very similar to that of line charts. You can use instructions from the [Line](/Demos/WidgetsGallery/Demo/Charts/Line/) demo description to configure Spline charts. To display a Spline, Stacked Spline, or Full-Stacked Spline series, set the [type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#type) property to one of these types. 
```

**Review by vladaskorohodova — APPROVED** [review]

**Pull request merged**